### DISCUSSION TITLE
geist-ui-svelte Example project

### DISCUSSION THREAD

**Discussion by ieedan** [ieedan/geist-ui-svelte#85] [Show and tell]
@SikandarJODD you can use this discussion for the example project

**Comment by SikandarJODD** [comment]
I will create a simple todo for using gesit-ui-svelte

**Reply by ieedan** [reply]
I think I am gonna build something as well.

Let me know about any issues you have with it lets just post them here and then we can open issues off of them.

**Reply by ieedan** [reply]
Also just as a rule I think we should make sure we are only using the documentation on the website to complete this and we aren't looking at the source.

**Comment by ieedan** [comment]
Add tabs for pnpm and yarn for installation commands

I use pnpm so this would be really nice so I can just copy paste

**Comment by ieedan** [comment]
Add `<slot/>` at the bottom of the +layout.svelte example

**Reply by ieedan** [reply]
complete

**Comment by ieedan** [comment]
Need to add a dependency for TailwindCSS at version 3.4.0 otherwise the library will break

**Reply by ieedan** [reply]
complete

**Comment by ieedan** [comment]
Add a `grow` attribute to elements like input that have a configurable width but no class

**Comment by SikandarJODD** [comment]
This is the Example : https://geist-example.vercel.app i build 
Issues i faced :
1. Button <code>width</code> should be increase like w-full, w-fit ...classes 
2. Input width should be flexible like width wise..
3. I think i should add <code>/examples</code> tab in geist-ui to get build in components like card n ll
4. i dont know how to add **Dark Mode** to website like i didn't find how to add it on website 
5. More Examples would help users to know more about Usage Stuff

**Reply by ieedan** [reply]
Sveltekit dark mode is just a pain in the but. You could rip what we use here for now.

An examples tab maybe under guide would be good.

`Input width should be flexible like width wise..`

What does this mean?

Same with the button width^

**Reply by SikandarJODD** [reply]
while i was using Input tag ...the width of input is fixed : https://geist-ui-svelte.dev/components/input 
Example : 
![image](https://github.com/ieedan/geist-ui-svelte/assets/93428946/bed9baeb-c0f2-4ef0-a404-fb50c4863220)

we can add width attribute 
```
<Input width={30} type='text'/>
```

**Reply by SikandarJODD** [reply]
Same goes with button too....

**Reply by ieedan** [reply]
RIght now the input has a width attribute as well as the button. Both are in docs

**Comment by ieedan** [comment]
Right now the Skeleton container is broken the loading attribute doesnt toggle the hidden state properly

**Comment by ieedan** [comment]
Add colors to card. Include text colors

**Comment by ieedan** [comment]
Make Text default value inherit

**Comment by ieedan** [comment]
Option to disable header transparency effect

**Comment by ieedan** [comment]
I built https://[echo-chamber-omega.vercel.app](https://echo-chamber-omega.vercel.app/)/

Gonna go fix stuff now

**Comment by ieedan** [comment]
@SikandarJODD I'm gonna get lunch then I'm gonna start opening issues
